samples/uriapp/src/main/java/com/facebook/samples/uriapp/MainActivity.java

```diff
@@ -36,6 +37,18 @@ protected void onCreate(Bundle savedInstanceState) {
     setContentView(R.layout.activity_main);
 
     mSimpleDraweeView = (SimpleDraweeView) findViewById(R.id.simple_drawee_view);
+    mSimpleDraweeView.setOnClickListener(new View.OnClickListener() {
+      @Override
+      public void onClick(View v) {
+        AnimatedDrawableSupport animatable = (AnimatedDrawableSupport)mSimpleDraweeView.getController().getAnimatable();
```

Comment:
This is not necessarily an `AnimatedDrawableSupport`, it can also be an `AnimatedDrawable`. You should cast this to `AbstractAnimatedDrawable` instead.